devel/nuweb/Portfile

```diff
@@ -1,30 +1,30 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           makefile 1.0
 
 name                nuweb
-version             1.58
+version             1.62
 categories          devel tex textproc print
-platforms           darwin
 maintainers         nomaintainer
 license             BSD
 
 description         a literate programming system compatible with any language
 long_description    {*}${description}
 
-homepage            http://nuweb.sourceforge.net/
+homepage            https://nuweb.sourceforge.net
 master_sites        sourceforge:project/${name}/
 
-patchfiles          patch-Makefile.diff
+checksums           rmd160  41d4ae2b57dd876af199b557be3f7dca0c46c616 \
+                    sha256  255a8f62460f5c14f4c4b356b96e0357a37a6652ae0584064f8e9fae19e953ae \
+                    size    210923
 
-checksums           rmd160  0c32ae064d742ff32bde14ab4a6a269c52772b84 \
-                    sha256  fe909b703c707c04f778c3d3d2f1eed2e10034f7025d292dd9ae041feb88a160
+patchfiles          patch-Makefile.diff
 
 use_configure       no
 
 build.target        nuweb
-build.env-append    CC=${configure.cc} \
-                    "CFLAGS=${configure.cflags} [get_canonical_archflags cc] -std=gnu89 -Wno-return-type"
+build.env-append    "CFLAGS=${configure.cflags} [get_canonical_archflags cc] -std=gnu89 -Wno-return-type"
```

Comment:
`CFLAGS` should also be set by the `makefile` PG as far as I can tell, removing this doesn't work?

Reply:
Well, there are some custom flags added here, I guess there was a reason for these?

Reply:
perhaps for the other ones, but CFLAGS should be set by the `makefile` PG just as `CC`. You only remove the latter, I am asking why not the `CFLAGS`.

Reply:
I will try dropping flags.

Reply:
OK we are good without flags.